pipeline/metadata/maxmind.py

```diff
@@ -27,12 +28,11 @@ def _maxmind_reader(filepath: str) -> geoip2.database.Reader:
 
   # MaxMind Reader will only take a filepath,
   # so we need to write the file to local disk
-  disk_filename = os.path.join('/tmp', os.path.basename(filepath))
-  with open(disk_filename, 'wb') as disk_file:
+  with tempfile.NamedTemporaryFile() as disk_file:
     disk_file.write(f.read())
```

Comment:
This will fully read the file before starting to write.

No need to change, but I believe using the utilities at https://docs.python.org/3/library/shutil.html would write as you read, without requiring for the entire file to be in memory and perhaps be faster.

Reply:
The files in question here are ~50mb so it's not too essential, but good thought.